Helper/Payment.php

```diff
@@ -311,7 +311,7 @@ public function getConfigurableProductBackordersOptions(Product $product = null)
     public function validateVirtual()
     {
         if ($this->quote->getIsVirtual() && !$this->quote->getCustomerIsGuest()) {
-            $countryId = self::VALIDATE_COUNTRY;
+            $countryId = '';
```

Comment:
why are we not adding default value instead keeping it empty to start with? what happens if it never goes in for loop?

Reply:
Yep, it is better to assign to a default value. Added `DEFAULT_REGION`